### PR TITLE
Keep plugin configs in common engine configuration

### PR DESCRIPTION
**Pull request by sashayakovtseva**
## Description of the Pull Request (PR):

Following https://github.com/sylabs/singularity/pull/4495#discussion_r329519096, plugin configurations are now stored in engine common config. 

#### Before submitting a PR, make sure you have done the following:

- Read the [Guidelines for Contributing](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](https://github.com/sylabs/singularity/blob/master/CHANGELOG.md) if necessary according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added tests to validate this PR and tested this PR locally with a `make testall`
- Based this PR against the appropriate branch according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](https://github.com/sylabs/singularity/blob/master/CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

File: pkg/runtime/engine/config/config_linux.go [hunk 1]
```diff
@@ -0,0 +1,133 @@
+package config
```
Comment (WestleyK) [inline pkg/runtime/engine/config/config_linux.go:6]: Missing copyright header here

```suggestion
// Copyright (c) 2019, Sylabs Inc. All rights reserved.
// This software is licensed under a 3-clause BSD license. Please consult the
// LICENSE.md file distributed with the sources of this project regarding your
// rights to use or distribute this software.

package config
```

File: pkg/runtime/engine/config/config.go [hunk 2]
```diff
@@ -5,14 +5,39 @@
```
Comment (WestleyK) [inline pkg/runtime/engine/config/config.go:5]: Should also update the copyright year in this file, `2018-2019`

**Review by WestleyK — COMMENTED** [review]
Looks fine to me, just suggest adding/updating the copyright header for consistency.

File: pkg/runtime/engine/config/config_linux.go (hunk 1 above)
Comment (sashayakovtseva) [reply pkg/runtime/engine/config/config_linux.go:6]: thanks!

**Comment by WestleyK** [comment]
Why was this closes? Is it not necessary anymore?

**Comment by sashayakovtseva** [comment]
Yes, we discussed with Cedric that FUSE should be a separate plugin and this PR is useless

**Pull request closed without merging**